documentation/book/proc-resizing-persistent-volumes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-resizing-persistent-volumes-{context}']
+= Resizing persistent volumes
+
+This procedure describes how to increase the size of persistent volumes used by existing {ProductName} cluster.
+This procedure can be applied to clusters using a single persistent volumes as well as to clusters using multiple persistent volumes in a JBOD storage.
+
+NOTE: {ProductPlatformName} currently supports only increasing size of persistent volumes.
+Shrinking of volumes is currently not possible.
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with support for volume resizing
+* A running Cluster Operator
+* A storage class which supports volume expansion
+* A Kafka cluster using persistent volumes created using given storage class
```

Comment:
Should we say more about 'given storage class'?

Reply:
This just refers to the storage class from the line above - i.e. a one which supports volume expansion. If there is some better wording, please suggest.